app/actions/capture.js

```diff
@@ -39,6 +43,10 @@ export function setCaptures (captures, slug, username) {
   return { type: SET_CAPTURES, captures, slug, username };
 }
 
+export function markPending (pokemon, slug, username) {
+  return { type: MARK_PENDING, captured: false, pokemon, slug, username };
```

Comment:
once we're hard-coding `false` in the reducer, we can remove `captured: false` from here